starter.js

```diff
@@ -0,0 +1,43 @@
+$(document).ready(function() {
+    $('#action').click(function() {
+	//DO NOT CHANGE CODE BELOW
+        var num1 = parseInt($('#num1').val());
+        var num2 = parseInt($('#num2').val());
+        clear();
+        addNumbers(num1, num2);
+        var result = multiplyNumbers(num1, num2);
+        $('#result-mult').text(result);
+	//DO NOT CHANGE CODE ABOVE
+
+    });
+
+	/*
+	 Below this comment, create a function 
+	 named addNumbers, which accepts two parameters.
+	 The function should add the two parameters together
+	 and write the result to the element with the id
+	 result-add
+	*/
+	function addNumbers(num1, num2) {
+		$("#result-add").html(num1 + num2);
+	}
+	/*
+	 Below this comment, create a function
+	 named multiplyNumbers, which accepts two parameters.
+	 The function should multiply the two parameters together
+	 and return the result.
+	*/
+	function multiplyNumbers(num1, num2){
+		$("#result-mult").html(num1 * num2);
```

Comment:
this function is supposed to multiply num1 and num2 and **return** the result - not write it. So I would do something like:
let num3 = num1 * num2;
return num3;